dwave/gate/circuit.py

```diff
@@ -220,6 +230,57 @@ def bits(self) -> ClassicalRegister:
 
         return bit_reg
 
+    @property
+    def state(self) -> Optional[NDArray]:
+        """The resulting state after simulating the circuit."""
+        if self._state is None and self._density_matrix is not None:
+            warnings.warn("State is mixed. Use 'Circuit.density_matrix' to access.")
```

Comment:
is this using the right stacklevel? I think you might want `stacklevel=2`

Reply:
Yes, you're right. Should be `stacklevel=2`.